index.js

```diff
@@ -67,6 +67,15 @@ let cachedRecords;
 let credentials;
 
 /**
+ * Maps between passed in user name and internal user name.
+ *
+ * @type {Object}
+ *
+ * @private
+ */
+let usernameMap;
```

Comment:
Can we use `cachedUsernames` instead?
Just trying to follow the same pattern we used for other similar variables.